src/Native/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -802,8 +815,9 @@ bool QueryCacheSize()
     }
 
 #elif defined(_WASM_)
-    // Processor cache size not available on WebAssembly
-    success = false;
+    // Processor cache size not available on WebAssembly, but we can't start up without it, so pick a reasonable value
```

Comment:
Should there be a WASMTODO to figure out what not having it means? (Not a GC person, so can't tell if this would be okay to ship with eventually.)

Reply:
On closer inspection, we don't actually use this value on Unix at all (and the value I'm currently setting just gets stomped). If we did use it, it would affect the size of gen0. Switched this to assign to the right variable and use the GC's default value (256KB).